tests/test_databinder.py

```diff
@@ -398,7 +398,7 @@ def run_query(input_filenames=None):
     Query: !UprootRaw |
                 [{"treename": "nominal"}]
   - Name: ttH4
-    Dataset: !Rucio user.kchoi:user.kchoi.fcnc_tHq_ML.ttH.v11
+    Dataset: !Rucio user.kchoi:user.kchoi.fcnc_tHq_ML.ttH.v112
```

Comment:
Is this particular change necessary?

Reply:
```
  - Name: ttH3
    RucioDID: user.kchoi:user.kchoi.fcnc_tHq_ML.ttH.v11
    Query: !UprootRaw |
                [{"treename": "nominal"}]
  - Name: ttH4
    Dataset: !Rucio user.kchoi:user.kchoi.fcnc_tHq_ML.ttH.v11
    Query: !UprootRaw '[{"treename": "nominal"}]'
```
These are the same samples, so the test case was raising a Duplicate Samples Error.

Reply:
Ah :-)